docs/adr/passthrough-rpc.md

```diff
@@ -0,0 +1,105 @@
+# Passthrough RPC in Ripple 
+
+ 
+
+## Problem 
+
+Some firebolt APIs can be completely encapsulated in an extension/plugin. The open source Ripple provides no middleware logic, composition, or data mapping. For these APIs, we should avoid as much (boilerplate) code in Ripple as possible. 
+
+This can be done as configuration instead of code. 
+
+## Solution 
+
+Support config for Ripple in the device manifest file
+
+```
+{ 
```

Comment:
Feel we can reuse the connections and endpoints to suppport for multiple proxied rpc methods 

```suggestion
{ 
"endpoints": {
  "thunder"  : {
      "url": "ws://127.0.01:9992/jsonrpc",
      "protocol": "websocket"
    }
}
```